### DISCUSSION TITLE
ERROR COMING in Github Account..

### DISCUSSION THREAD

**Discussion by Ambarish-224** [community/community#41384] [Projects and Issues]
The error is :- Your github account has been flagged..

How to overcome this error..

Please help to resolve my Query..🙏🙏

**Comment by jfullstackdev** [comment]
any unusual activity you had with your account ? this happens when the user violated some of the rules of GitHub

you can also open a ticket here if you feel you are not violating any of those rules

https://support.github.com/contact/account

**Comment by jinyoung4478** [comment]
Yeah, I think you have to visit here. https://support.github.com/contact/account
It seems to be a problem caused by long-term disconnection.
